lib/pages/profile_page/community_list.dart

```diff
@@ -8,11 +8,25 @@ class CommunityListWidget extends StatelessWidget {
       {super.key,
       required this.user,
       required this.communityLength,
-      required this.token});
+      required this.token,
+      required this.currentEpoch});
 
   final User user;
   final int? communityLength;
   final String token;
+  final int currentEpoch;
+
+  computeStreak(firstStreakEpoch, lastStreakEpoch) {
+    firstStreakEpoch = int.parse(firstStreakEpoch);
+    lastStreakEpoch = int.parse(lastStreakEpoch);
+    if (currentEpoch - 86400000 < lastStreakEpoch) {
```

Comment:
Can you put the `86400000` in a constant so it's easy to tell that it's a day in milliseconds?